driver/compile.go

```diff
@@ -28,7 +27,7 @@ type Config struct {
 // and compiles it into a runnable flowgraph, returning a
 // proc.MuxOutput that which brings together all of the flowgraphs
 // tails, and is ready to be Pull()'d from.
-func Compile(ctx context.Context, program ast.Proc, zctx *resolver.Context, reader zbuf.Reader, cfg Config) (*MuxOutput, error) {
+func Compile(ctx context.Context, program ast.Proc, zctx *resolver.Context, readers []zbuf.Reader, cfg Config) (*MuxOutput, error) {
```

Comment:
I'd prefer to do this:
```suggestion
func Compile(ctx context.Context, program ast.Proc, zctx *resolver.Context, cfg Config, readers ...zbuf.Reader) (*MuxOutput, error) {
```

Reply:
I've removed this change to the signature of driver.Compile.